<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Main View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftVerticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <!-- Tab 1: ‡∏™‡∏¥‡∏ô‡∏Ñ‡πâ‡∏≤ -->
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string>‡∏™‡∏¥‡∏ô‡∏Ñ‡πâ‡∏≤</string>
         </attribute>
         <layout class="QVBoxLayout" name="tab1Layout">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>18</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>‡∏™‡∏¥‡∏ô‡∏Ñ‡πâ‡∏≤</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTableView" name="tableView_2"/>
             </item>
             <item>
              <widget class="QPushButton" name="addButton">
               <property name="text">
                <string>‚ûï ‡πÄ‡∏û‡∏¥‡πà‡∏°‡∏™‡∏¥‡∏ô‡∏Ñ‡πâ‡∏≤</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>‡πÄ‡∏•‡∏Ç‡∏ó‡∏µ‡πà‡∏Ñ‡∏≥‡∏™‡∏±‡πà‡∏á‡∏ã‡∏∑‡πâ‡∏≠</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOrderId"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOrder">
            <property name="text">
             <string>‡∏™‡∏±‡πà‡∏á‡∏ã‡∏∑‡πâ‡∏≠‡∏™‡∏¥‡∏ô‡∏Ñ‡πâ‡∏≤</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <!-- End of tab1 -->

        <!-- Tab 2: ‡πÉ‡∏ö‡∏™‡πà‡∏á‡∏Ç‡∏≠‡∏á -->
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>‡πÉ‡∏ö‡∏™‡πà‡∏á‡∏Ç‡∏≠‡∏á</string>
         </attribute>
         <layout class="QVBoxLayout" name="tab2Layout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>‡πÄ‡∏•‡∏Ç‡∏ó‡∏µ‡πà‡πÉ‡∏ö‡∏™‡πà‡∏á‡∏Ç‡∏≠‡∏á</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>‡∏Ñ‡πâ‡∏ô‡∏´‡∏≤</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="title">
             <string>‡∏£‡∏≤‡∏¢‡∏Å‡∏≤‡∏£‡πÉ‡∏ö‡∏™‡πà‡∏á‡∏Ç‡∏≠‡∏á</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QTableView" name="tableView"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <!-- End of tab2 -->

        <!-- Tab 3: ‡πÉ‡∏ö‡∏Å‡∏≥‡∏Å‡∏±‡∏ö‡∏†‡∏≤‡∏©‡∏µ -->
        <widget class="QWidget" name="tab3">
         <attribute name="title">
          <string>‡πÉ‡∏ö‡∏Å‡∏≥‡∏Å‡∏±‡∏ö‡∏†‡∏≤‡∏©‡∏µ</string>
         </attribute>
         <layout class="QVBoxLayout" name="tab3Layout">
          <item>
           <widget class="QLabel" name="label_invoice_number">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>‡πÄ‡∏•‡∏Ç‡∏ó‡∏µ‡πà‡πÉ‡∏ö‡∏Å‡∏≥‡∏Å‡∏±‡∏ö‡∏†‡∏≤‡∏©‡∏µ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_invoice_number"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_search_invoice">
            <property name="text">
             <string>‡∏Ñ‡πâ‡∏ô‡∏´‡∏≤</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_invoices">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="title">
             <string>‡∏£‡∏≤‡∏¢‡∏Å‡∏≤‡∏£‡πÉ‡∏ö‡∏Å‡∏≥‡∏Å‡∏±‡∏ö‡∏†‡∏≤‡∏©‡∏µ</string>
            </property>
            <layout class="QVBoxLayout" name="groupBoxInvoicesLayout">
             <item>
              <widget class="QTableView" name="tableView_invoices"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <!-- End of tab3 -->

        <!-- Tab 4: ‡∏Ñ‡∏≥‡∏ô‡∏ß‡∏ì‡∏ï‡πâ‡∏ô‡∏ó‡∏∏‡∏ô -->
       <widget class="QWidget" name="tab4">
        <attribute name="title">
        <string>‡∏Ñ‡∏≥‡∏ô‡∏ß‡∏ì‡∏ï‡πâ‡∏ô‡∏ó‡∏∏‡∏ô</string>
        </attribute>
        <layout class="QVBoxLayout" name="tab4Layout">
        <item>
        <widget class="QGroupBox" name="groupBox">
            <property name="font">
            <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            </font>
            </property>
            <property name="title">
            <string>‡∏Ñ‡∏≥‡∏ô‡∏ß‡∏ì‡∏ï‡πâ‡∏ô‡∏ó‡∏∏‡∏ô‡∏™‡∏¥‡∏ô‡∏Ñ‡πâ‡∏≤</string>
            </property>
            <layout class="QFormLayout" name="groupBoxLayout">
            <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignVCenter</set>
            </property>

            <item row="0" column="0">
            <widget class="QLabel" name="label_product">
            <property name="text">
                <string>‡∏™‡∏¥‡∏ô‡∏Ñ‡πâ‡∏≤:</string>
            </property>
            </widget>
            </item>
            <item row="0" column="1">
            <widget class="QComboBox" name="comboBox_products"/>
            </item>

            <item row="1" column="0">
            <widget class="QLabel" name="label_quantity">
            <property name="text">
                <string>‡∏à‡∏≥‡∏ô‡∏ß‡∏ô:</string>
            </property>
            </widget>
            </item>
            <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_quantity">
            <property name="minimum">
                <number>1</number>
            </property>
            <property name="maximum">
                <number>10000</number>
            </property>
            <property name="value">
                <number>1</number>
            </property>
            </widget>
            </item>

            <item row="2" column="0">
            <widget class="QLabel" name="label_sale_price">
            <property name="text">
                <string>‡∏£‡∏≤‡∏Ñ‡∏≤‡∏Ç‡∏≤‡∏¢‡∏ï‡πà‡∏≠‡∏´‡∏ô‡πà‡∏ß‡∏¢ (‡∏ö‡∏≤‡∏ó):</string>
            </property>
            </widget>
            </item>
            <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_sale_price"/>
            </item>

            <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_calculate_cost">
            <property name="text">
                <string>üìä ‡∏Ñ‡∏≥‡∏ô‡∏ß‡∏ì‡∏ï‡πâ‡∏ô‡∏ó‡∏∏‡∏ô</string>
            </property>
            </widget>
            </item>

            <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_result">
            <property name="text">
                <string>‡∏ú‡∏•‡∏•‡∏±‡∏û‡∏ò‡πå: </string>
            </property>
            <property name="font">
                <font>
                <pointsize>14</pointsize>
                <bold>true</bold>
                </font>
            </property>
            </widget>
            </item>
            </layout>
        </widget>
        </item>
        </layout>
        </widget>
        <!-- End of tab4 -->
        
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightPanelLayout">
      <item>
       <widget class="QGroupBox" name="groupBoxRight">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="title">
         <string>‡∏™‡∏£‡∏∏‡∏õ‡∏≠‡∏≠‡∏Å‡∏£‡∏≤‡∏¢‡∏á‡∏≤‡∏ô</string>
        </property>
        <layout class="QVBoxLayout" name="groupBoxRightLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>‡∏ß‡∏±‡∏ô‡πÄ‡∏£‡∏¥‡πà‡∏°‡∏ï‡πâ‡∏ô</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCalendarWidget" name="calendarWidget"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>‡∏ß‡∏±‡∏ô‡∏™‡∏¥‡πâ‡∏ô‡∏™‡∏∏‡∏î</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCalendarWidget" name="calendarWidget_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>‡∏≠‡∏≠‡∏Å‡∏£‡∏≤‡∏¢‡∏á‡∏≤‡∏ô</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <widget class="QLabel" name="datetimeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>DateTime goes here</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
